libraries/postgres/src/Database.js

```diff
@@ -391,9 +391,10 @@ class Database {
         FROM pg_database 
         WHERE datname = current_database()`);
       const collation = res.rows[0].collation;
-      if (collation !== 'en_US.UTF8') {
+      if (collation !== 'en_US.utf8' && collation !== 'en_US.UTF8') {
         throw new Error(
-          `Postgres database must have default collation en_US.UTF8; this database is using ${collation}.`);
+          `Postgres database must have default collation en_US.utf8 or en_US.UTF8; this database is using ${collation}.`,
+        );
```

Comment:
This should probably be left out of the PR. Rebasing on master might do the trick.

Reply:
Ah, has an equivalent fix landed on master?

Reply:
Ah. Forget what I said. I just realized it was simply a change to an error message :)

Reply:
I wonder what's behind this difference in case, but ultimately I think the two mean the same thing, so this looks good.

Reply:
Yeah, I'm not sure. The test failed for me locally using the postgres docker image because the value was `en_US.utf8` instead of the expected `en_US.UTF8`. If you don't have the same issue with the postgres docker container, perhaps something in my environment is influencing it. I have `LC_CTYPE=UTF-8` in my environment, but that is already capitalised. In any case, I agree it should be safe to assume `en_US.utf8` and `en_US.UTF8` refer to the same encoding. I think the official name includes the hyphen (`UTF-8`) so I think neither of the values are technically correct. But I digress... :-)